File: MODUL 6 PEMROGRAMAN 1/PRAK601-2310817120002-SITI RATNA DWINTA SARI.py
baris, kolom = map(int, input().split())
matriks = list(map(int, input().split()))
matriks = [matriks[i:i+kolom] for i in range(0, len(matriks), kolom)]

for i in range(baris):
    for j in range(kolom):
        print(matriks[i][j], end=" ")
    print()

File: MODUL 6 PEMROGRAMAN 1/PRAK601-2310817120002-SITIRATNADWINTASARI.c
#include <stdio.h>
int main() {
    int baris, kolom;
    scanf("%d %d", &baris, &kolom);
    int matriks[baris][kolom];
    for (int i = 0; i < baris; i++) {
        for (int j = 0; j < kolom; j++) {
            scanf("%d", &matriks[i][j]);
        }
    }
    for (int i = 0; i < baris; i++) {
        for (int j = 0; j < kolom; j++) {
            printf("%d ", matriks[i][j]);
        
        }
        printf("\n");
    }
}